e2e/test/scenarios/embedding/embedding-dashboard.html

```diff
@@ -0,0 +1,11 @@
+<script src="http://localhost:4000/app/iframeResizer.js"></script>
```

Comment:
Add a new HTML file for testing embedding because I need to import `http://localhost:4000/app/iframeResizer.js`.

Thess scripts follow our [doc for static embedding.](https://www.metabase.com/docs/latest/embedding/static-embedding#resizing-dashboards-to-fit-their-content)